tests/test.py

```diff
@@ -512,7 +512,7 @@ def run_terraform(*, image: str, final: bool = False):
             '/bin/bash -c "terraform init -backend=false && terraform validate"',
             1,
         ),  # This tests the bring-your-own TERRAFORM_VERSION hook, regardless of the built-in security tools
-        # It succeeds because only terraform/hooks/secure_1_1/secure.tf is tested, and it fails because it requires a version of terraform newer then
+        # It succeeds because only terraform/hooks/secure_1_3/secure.tf is tested, and it fails because it requires a version of terraform newer then
```

Comment:
Please confirm that this comment is still true.  Look at the `tests` list and the `exec_tests` below

Reply:
```suggestion
        ),  # This tests the bring-your-own TERRAFORM_VERSION hook (40-), regardless of the built-in security tools (DISABLE_SECURITY=true)
        # It fails because it ignores the 50- terraform due to AUTODETECT=false, and the v_0_14_dir files fail given the version of TERRAFORM_VERSION specified above
```